src/index.ts

```diff
@@ -2,8 +2,8 @@ console.warn('REAtom still work in progress, do not use it in production')
 
 type NodeId = string
 
-type ActionType = string
-type ActionTypesDictionary = Record<ActionType, true>
+type Dictionary<T> = { [key: string]: T }
+type ActionTypesDictionary = Record<NodeId, true>
```

Comment:
` type ActionTypesDictionary = Record<ActionType, true>` was right

Reply:
But you may replace Record by Dictionary

Reply:
mean like `type ActionTypesDictionary = Dictionary<bool>`?